Bricks/Gnuplot/0.0.1.toml

```diff
@@ -0,0 +1,9 @@
+
+[brick]
+name = "Gnuplot"
+version = "0.0.1"
```

Comment:
Was this version number inherited from the original source? If not, I'd suggest starting with `0.1.0` as initial version number.

Reply:
This was inherited from the source, I used 'mason publish' to do this and it just grabs it from the source.